### ISSUE TITLE
Shared Video not playing in IOS browser, just showing the thumbnail

### ISSUE THREAD

**Issue by sree-1621** [jitsi/jitsi-meet#14521]
### Steps to reproduce:

1.  Join as a participant from Iphone 
2.  Share some public youtube link in share video dialog
3.  The video wont play and end call button is not working after that.

### Expected behavior:

Actually the video should play in Iphone not just thumbnail

### Actual behavior:
Now it is simply showing the thumbnail in the player, nothing more than that and afterwards not able to click the end call button also. But in android the shared video is playing but in IOS it is not

### Question

How we can solve it?!

**Comment by saghul** [comment]
Are you sharing a youtube video or a normal video file?

**Comment by Calinteodor** [comment]
I tested this issue on latest beta and I am unable to reproduce the reported issue. At least, YouTube videos work. 
What device are you using? Can share some logs?

**Comment by nitinkushwaha17** [comment]
@Calinteodor I am also facing the same issue on IPad (both chrome and safari).
Shared video doesn't play on meet.jit.si but a thumbnail is shown.

**Comment by saghul** [comment]
Hold on, this is in the browser and not the apps?

**Comment by nitinkushwaha17** [comment]
For me app works fine on IPad.
Yes the problem is in browser

**Comment by sree-1621** [comment]
> Are you sharing a youtube video or a normal video file?

I tried sharing the youtube video

**Comment by sree-1621** [comment]
And I want to say that when we toggle the **audio button** in the browser, the video starts playing in iphone too, I guess there is something related to that audio button is affecting the video display

**Comment by Calinteodor** [comment]
I think this is something specific to mobile browsers when talking about media. There needs to be a user gesture for it to start playing.

**Comment by Calinteodor** [comment]
Sorry. I think I didn't understand exactly your last comment. You are talking about the audio mute button from the toolbox menu or the audio mute button from the video player? And when you press it, the video starts playing?

**Comment by sree-1621** [comment]
No, the Audio Button from the toolbox is affecting the video . When i clicked the audio button, the video started playing.

**Comment by Calinteodor** [comment]
So I tried the following:
1. Join from mobile Safari browser.
2. Tap Share Video button in the Overflow Menu.
3. Paste Youtube link.
4. Video appears in the conference.
5. Press Play video starts.

Are these the correct steps? The second time I shared a video I saw that the YouTube video player menu was covered by Jitsi Toolbox menu, but I could tap on the video and it paused/played, but maybe this is the case because it's a YouTube video. After that I put the phone in landscape and back on portrait and the menus separated from each other.
